sycl/include/CL/sycl/detail/accessor_impl.hpp

```diff
@@ -70,10 +72,8 @@ class AccessorImplHost {
         MElemSize(ElemSize), MOffsetInBytes(OffsetInBytes),
         MIsSubBuffer(IsSubBuffer) {}
 
-  ~AccessorImplHost() {
-    if (BlockingEvent)
-      BlockingEvent->setComplete();
-  }
+  ~AccessorImplHost();
```

Comment:
Why do you need to specify this?

Reply:
To let compiler know that we have "custom" destructor which is defined in sycl/source/detail/accessor_impl.cpp .

Reply:
Ah sorry for the noise, I read `~AccessorImplHost() {};` :-(